cypress/e2e/item/duplicate/listDuplicateItem.cy.ts

```diff
@@ -0,0 +1,54 @@
+import { getParentsIdsFromPath } from '@/utils/item';
+
+import { HOME_PATH, buildItemPath } from '../../../../src/config/paths';
+import {
+  ITEM_MENU_DUPLICATE_BUTTON_CLASS,
+  buildItemMenu,
+  buildItemMenuButtonId,
+  buildItemsTableRowIdAttribute,
+} from '../../../../src/config/selectors';
+import ITEM_LAYOUT_MODES from '../../../../src/enums/itemLayoutModes';
+import { SAMPLE_ITEMS } from '../../../fixtures/items';
+
+const duplicateItem = ({ id }: { id: string }) => {
```

Comment:
This function is duplicated (how ironic 😉 ) between the grid and list files. Maybe we could put in in a common file for re-use ?